core/sql/executor/HBaseClient_JNI.cpp

```diff
@@ -434,10 +429,7 @@ HBC_RetCode HBaseClient_JNI::releaseHTableClient(HTableClient_JNI* htc)
       tsRecentJMFromJNI = JavaMethods_[JM_REL_HTC].jm_full_name;
       jenv_->CallVoidMethod(javaObj_, JavaMethods_[JM_REL_HTC].methodID, j_htc);
       if (jenv_->ExceptionCheck()) {
-         getExceptionDetails();
-         logError(CAT_SQL_HBASE, __FILE__, __LINE__);
-         logError(CAT_SQL_HBASE, "HBaseClient_JNI::releaseHTableClient()", getLastError());
-         jenv_->PopLocalFrame(NULL);
```

Comment:
Did you mean to delete the jenv_->PopLocalFrame call here?

Reply:
Yes. I will add it back in my next check-in